pkgs/development/python-modules/mayavi/default.nix

```diff
@@ -1,37 +1,43 @@
-{ stdenv, fetchPypi, buildPythonPackage
-, wxPython, pygments, numpy, vtk, traitsui, envisage, apptools
-, nose, mock
-, isPy3k
-}:
+{ lib, pythonPackages, wrapQtAppsHook }:
```

Comment:
Do *not* pass in `pythonPackages` in `python-modules`; always have parameters for the individual packages.

Reply:
Sorry, this is how the other Python packages using `pkgs.libsForQt5.callPackage` are currently structured.  Fixed.

Reply:
@FRidh: is there a particular reason why this is best practice?

Reply:
it's good practice to be explicit about inputs.